CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+---
+
+# Contributing to Diffcessible
+
+Thank you for your interest in contributing to Diffcessible! This guide outlines how to contribute to the project effectively and make the most out of your collaboration experience.
+
+## Getting Started
+
+1. **Set Up the Project:** Make sure you have forked the repository and set up the project on your local machine. Follow the installation instructions in the README.md file.
+
+2. **Understand the Code:** Spend some time getting familiar with the codebase. It's important to have a good grasp of how the project is structured before starting to contribute.
+
+## Claiming an Issue
+
+- **Find an Issue:** Start with "good first issues" to find tasks suitable for newcomers.
+- **Claim an Issue:** To avoid duplicate efforts, you need to be assigned to an issue before working on it. To be assigned:
+  - Ensure you can build the project successfully.
+  - Comment on the issue thread with a screenshot showing the `--help` page of the project on your computer.
+  - Ask to be assigned to the issue.
+- **Getting Help:** If you need further guidance on the issue you're working on, don't hesitate to ask. We're here to help!
+
+## Asking for Help
+
+Encountering challenges is a normal part of the process. If you need help:
+
+- **General OCaml and Tooling Help:** Join our Discord server or visit the OCaml Discuss forum. Look for the "#outreachy" and "#beginner" channels on Discord for guidance.
+- \*Project-Specific Help:\*\* Feel free to ask for help within the GitHub issue threads or on our communication channels.
+
+## Contributing Guidelines
+
+- **Code Quality:** Write clean, understandable, and maintainable code. Follow the coding standards and conventions of the project.
```

Comment:
There's no coding standard and convention written down in the project and I'm not sure I'd want one. That's too complex to define and I don't want to ask people to read such a document and change their way of coding.

I think this section should be more concrete:
- Do not mention code quality.
- Mention OCamlformat and give the command to run it.
- Give the command to run the tests.

Reply:
Resolved